pkgs/data/themes/mojave/default.nix

```diff
@@ -1,24 +1,32 @@
-{ stdenv, fetchFromGitHub, gtk_engines, gtk-engine-murrine }:
+{ stdenv, fetchFromGitHub, fetchurl, gtk_engines, gtk-engine-murrine }:
 
 stdenv.mkDerivation rec {
   pname = "mojave-gtk-theme";
-  version = "2019-09-09";
+  version = "2019-12-12";
 
-  src = fetchFromGitHub {
-    owner = "vinceliuice";
-    repo = pname;
-    rev = version;
-    sha256 = "1qffh6jsvy61f29ymw1v9hpjnsvhqin19mp05cys1lnwc7y810zr";
-  };
+  srcs = [
+    (fetchFromGitHub {
+      owner = "vinceliuice";
+      repo = pname;
+      rev = version;
+      sha256 = "0d5m9gh97db01ygqlp2sv9v1m183d9fgid9n9wms9r5rrrw6bs8m";
+    })
+    (fetchurl {
+      url = "https://raw.githubusercontent.com/vinceliuice/Mojave-gtk-theme/images/macOS_Mojave_Wallpapers.tar.xz";
```

Comment:
Is that URL stable enough over time?

Reply:
I think so. It has never changed in the git repository.

![Screenshot from 2019-12-13 12-11-52](https://user-images.githubusercontent.com/1217934/70810067-d6dd7800-1da1-11ea-982c-50fff9ed94e3.png)

Reply:
This url will be stable https://github.com/vinceliuice/Mojave-gtk-theme/raw/11741a99d96953daf9c27e44c94ae50a7247c0ed/macOS_Mojave_Wallpapers.tar.xz

Reply:
> This url will be stable https://github.com/vinceliuice/Mojave-gtk-theme/raw/11741a99d96953daf9c27e44c94ae50a7247c0ed/macOS_Mojave_Wallpapers.tar.xz

Changed to this URL.